html/partials/packages.html.haml

```diff
@@ -23,7 +23,7 @@
                   %th.medium Vendor
                   %th Checksum
               %tbody
-                - packages.each do |package|
+                - packages.packages.each do |package|
```

Comment:
I know this is the code following the model but it just looks bad to me to have packages.packages, what do you think?